enterprise/cmd/repo-updater/shared/shared.go

```diff
@@ -91,14 +70,11 @@ func EnterpriseInit(
 	resetter := authz.MakeResetter(observationCtx, repoWorkerStore)
 
 	go goroutine.MonitorBackgroundRoutines(ctx, repoSyncWorker, userSyncWorker, resetter)
-
-	go startBackgroundPermsSync(ctx, permsSyncer, db)
-
-	return map[string]debugserver.Dumper{"repoPerms": permsSyncer}, enqueueRepoPermsJob
+	go watchAuthzProviders(ctx, db)
 }
 
-// startBackgroundPermsSync sets up background permissions syncing.
-func startBackgroundPermsSync(ctx context.Context, syncer *authz.PermsSyncer, db ossDB.DB) {
+// watchAuthzProviders sets up background permissions syncing.
```

Comment:
Should we also change the docstring?
```suggestion
// watchAuthzProviders sets up authz providers if config changes
```

Reply:
nice catch, thanks!